core/src/main/java/nl/nn/adapterframework/monitoring/Monitor.java

```diff
@@ -62,7 +64,7 @@
 
 	private MonitorManager manager = null;
 
-	private List<Trigger> triggers = new ArrayList<>();
+	private List<TriggerBase> triggers = new ArrayList<>();
```

Comment:
De elementen van deze lijst (en ook de meeste andere plekken) moeten van het type ITrigger zijn, in plaats van TriggerBase